src/clients/FlexbaseClient.Card.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Card } from '../models/Card/Card';
+import { Address } from '../models/Address/Address';
+import { CardOptions } from '../models/Card/CardOptions';
+import { FlexbaseClientBase } from './FlexbaseClient.Base';
+import { FlexbaseResponse } from '../models/FlexbaseResponse';
+
+interface QueryParameters {
+    searchTerm?: string;
+    status?: string;
+}
+
+interface CardsResponse extends FlexbaseResponse{
+    cards: Card[];
+}
+
+interface CardResponse extends FlexbaseResponse{
+    card: Card;
+}
+
+interface IssueCardForm {
+    cardType: string,
+    shipTo?: Address,
+    service?: string,
+}
+
+interface UpdateCardForm {
+    expensesTypes: {
+        amount: number,
+        groups: any,
+    };
+    notifyUse: boolean;
+    shipTo?: Address,
+    creditLimit: number,
+    cardName?: string,
+}
+
+export class FlexbaseClientCard extends FlexbaseClientBase {
+    private params({ searchTerm, status }: CardOptions = {}) {
+        const params: any = {};
+
+        if (typeof searchTerm === 'string') {
+            params.searchTerm = searchTerm;
+          }
+      
+          if (typeof status === 'string') {
+            params.status = status;
+          }
```

Comment:
The `CardOptions` model already establishes that both of these fields are nullable strings. Hence, instead of checking the type of `searchTerm` or `status`, we should be checking if they are `null`. So replace both of these checks with null checks.

Also, the indentation of the second check is off.

Reply:
something like
```
        if (searchTerm !== null) {
            params.searchTerm = searchTerm;
          }
```
?